src/Roslyn.Suppressions/Roslyn.Suppressions/SupportedSuppressionBuilder.g.cs

```diff
@@ -48,7 +48,7 @@ private static ImmutableArray<SuppressionDescriptor> GetDescriptors()
                 // If the number of diagnostic ids changes significantly or decreases when the analysers are updated,
                 // investigate! There may be a problem with code generator.
                 // *************************************************************************************************
-                // Number of unique diagnostic ids (C# and VB.NET): 500
+                // Number of unique diagnostic ids (C# and VB.NET): 504
```

Comment:
These rules were added in 10.2, so I guess we forgot to re-generate the file back then

https://github.com/SonarSource/sonar-dotnet-enterprise/releases/tag/10.2.0.105762